### PR TITLE
CASMCMS-8894 Include new image management workflow instructions for updating image labels

### PR DESCRIPTION
**Pull request by jsl-hpe**
# Description
IMS now allows administrators to decorate images with user supplied key/value strings. Included are examples for how to use the CLI in such a way to affect those changes on an image.
Relates to:
- [https://jira-pro.it.hpe.com:8443/browse/CASMCMS-8915](CASMCMS-8915) (merged to 1.6)

# Checklist
- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

### CODE REVIEW

File: operations/image_management/Image_Management_Workflows.md [hunk 1]
```diff
@@ -172,3 +173,54 @@ Mentioned in this workflow:
...
+    ```bash
+    cray ims images update a506a6f6-54d9-4e5a-9e8d-1fc052d62504 --metadata-operation set --metadata-value value --metadata-key key
+    ```
+
+1. (`ncn-mw#`) Administrators and downstream APIs may obtain the active record for a given image.
+
+    Image metadata information is also available via the `list` command for all images.
+
+    ```bash
+    cray ims images describe a506a6f6-54d9-4e5a-9e8d-1fc052d62504
```
Comment (dlaine-hpe) [inline operations/image_management/Image_Management_Workflows.md:209]: I think they want this broken into command and expected output chunks:
```bash
cray ims images describe a506a6f6-54d9-4e5a-9e8d-1fc052d62504
```

Expected output:

```toml
arch = "x86_64"
created = "2024-06-27T15:41:22.467177"
id = "a506a6f6-54d9-4e5a-9e8d-1fc052d62504"
[metadata]
key = "value"
```

File: operations/image_management/Image_Management_Workflows.md [hunk 2]
```diff
@@ -172,3 +173,54 @@ Mentioned in this workflow:
...
+    [metadata]
+    key = "value"
+    ```
+
+1. (`ncn-mw#`) Administrators may remove previously set image metadata.
+
+    Downstream APIs and Administrators using the CLI may affect these changes. During `--metadata-operation remove`, users may omit `--metadata-value` command line arguments.
+
+    ```bash
+    cray ims images update a506a6f6-54d9-4e5a-9e8d-1fc052d62504 --metadata-operation remove --metadata-key key
```
Comment (dlaine-hpe) [inline operations/image_management/Image_Management_Workflows.md:227]: Same here with splitting command / expected output.

**Review by dlaine-hpe — APPROVED** [review]

File: operations/image_management/Image_Management_Workflows.md [hunk 3]
```diff
@@ -172,3 +173,64 @@ Mentioned in this workflow:
...
+
+    Image metadata information is also available via the `list` command for all images.
+
+    ```bash
+    cray ims images describe a506a6f6-54d9-4e5a-9e8d-1fc052d62504
+    ```
+
+    Expected output:
+
+    ```bash
```
Comment (rustydb) [inline operations/image_management/Image_Management_Workflows.md]:
```suggestion
    ```toml
```

File: operations/image_management/Image_Management_Workflows.md [hunk 4]
```diff
@@ -172,3 +173,64 @@ Mentioned in this workflow:
...
+
+    Downstream APIs and Administrators using the CLI may affect these changes. During `--metadata-operation remove`, users may omit `--metadata-value` command line arguments.
+
+    ```bash
+    cray ims images update a506a6f6-54d9-4e5a-9e8d-1fc052d62504 --metadata-operation remove --metadata-key key
+    ```
+
+    Expected output:
+
+    ```bash
```
Comment (rustydb) [inline operations/image_management/Image_Management_Workflows.md]:
```suggestion
    ```toml
```

File: operations/image_management/Image_Management_Workflows.md [hunk 5]
```diff
@@ -172,3 +173,64 @@ Mentioned in this workflow:
...
+
+1. (`ncn-mw#`) Administrator identifies the image to add metadata information to.
+
+    Administrators may already know the image ID in question to label. If not, examining the existing images may be of help.
+
+    ```bash
+    cray ims images list
+    ```
+
+1. (`ncn-mw#`) Administrators may set a new label for an existing IMS Image
```
Comment (nrockershousen) [inline operations/image_management/Image_Management_Workflows.md]:
```suggestion
1. (`ncn-mw#`) Administrators may set a new label for an existing IMS image.
```

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**